tests/test_contrib_cern.py

```diff
@@ -37,6 +38,22 @@ def test_fetch_groups(app, example_cern):
                for group in ('Group{}'.format(i) for i in range(4, 6)))
 
 
+def test_fetch_extra_data(app, example_cern):
```

Comment:
Can you please add a test where the response does NOT contain some of the values too?
So we ensure that future changes will not break behaviour. Thanks!

Reply:
@ntarocco [here](https://github.com/inveniosoftware/invenio-oauthclient/pull/179/files#diff-25fc83081804f5e108a7dd26baf648beR57)